packages/maskbook/src/plugins/Gitcoin/UI/DonateDialog.tsx

```diff
@@ -217,7 +216,6 @@ export function DonateDialog(props: DonateDialogProps) {
         <div className={classes.root}>
             <InjectedDialog open={open} onClose={onClose} title={title} DialogProps={{ maxWidth: 'xs' }}>
                 <DialogContent>
-                    <EthereumStatusBar classes={{ root: classes.root }} />
                     <form className={classes.form} noValidate autoComplete="off">
```

Comment:
I have a question. You're removing those bars from all dialogs. But how can you click that global injected bar when a dialog is opened?

Reply:
Umm. I think the user will be unable to change the account if a dialog was opened. They should choose the correct account before open any dialogs.